R/tutorial.R

```diff
@@ -14,6 +14,7 @@
 #' @param name Base for file name to use for new `.Rmd` tutorial. Should consist
 #'   only of numbers, letters, `_` and `-`. We recommend using lower case.
 #' @param title The human-facing title of the tutorial.
+#' @param learner_friendly If `TRUE`, adds `learnr` to Imports instead of Suggests to install `learnr` alongside the tutorial package. Re-exports `run_tutorial()` to make it immediately available to tutorial package users.
```

Comment:
Did you have any other ideas for argument name? My thought is that it could be more factual / descriptive. Because who wouldn't want to be `learner_friendly`? It feels odd to default to something that seems learner-unfriendly.

A possible re-framing: instead of the new argument being `TRUE` / `FALSE` and learner friendly vs ?unfriendly?, it could take values "Suggests" vs. "Imports". That might suggest another name for the argument.

Reply:
Our thought process behind this is that `learner_friendly` would address two types of users:

1. FALSE (lightweight option) -- for package developers who want to include tutorials as documentation but not absorb extra dependencies
2. TRUE (heavyweight option) -- for tutorial-only packages (for educators, etc)

We wanted a function name that would make it clear what this option was meant for.  Perhaps using the default `lightweight = TRUE` could be relatively neutral (unlike `unfriendly`) and still evocative?  @angela-li perhaps you have some thoughts on this?